calliope/backend/subsets.py

```diff
@@ -196,14 +202,29 @@ def __is_equals_statement(imask_string):
     return imask
 
 
-def _combine_imasks(curr_imask, new_imask, _operator):
+def _translate_operator(_operator):
```

Comment:
This will be a lot [nicer in Python 3.10](https://docs.python.org/3.10/whatsnew/3.10.html#pep-634-structural-pattern-matching)